{"variants":[{"paths":["\/documentation\/pingonewallet\/walletexception\/localizederror-implementations"],"traits":[{"interfaceLanguage":"swift"}]}],"hierarchy":{"paths":[["doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet","doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletException"]]},"sections":[],"metadata":{"modules":[{"name":"PingOneWallet"}],"title":"LocalizedError Implementations","role":"collectionGroup","roleHeading":"API Collection"},"schemaVersion":{"minor":3,"patch":0,"major":0},"kind":"article","identifier":{"url":"doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletException\/LocalizedError-Implementations","interfaceLanguage":"swift"},"topicSections":[{"generated":true,"identifiers":["doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletException\/failureReason","doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletException\/helpAnchor","doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletException\/recoverySuggestion"],"anchor":"Instance-Properties","title":"Instance Properties"}],"references":{"doc://com.pingidentity.PingOneWallet/documentation/PingOneWallet/WalletException":{"type":"topic","identifier":"doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletException","abstract":[],"kind":"symbol","url":"\/documentation\/pingonewallet\/walletexception","role":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"WalletException"}],"navigatorTitle":[{"kind":"identifier","text":"WalletException"}],"title":"WalletException"},"doc://com.pingidentity.PingOneWallet/documentation/PingOneWallet/WalletException/failureReason":{"url":"\/documentation\/pingonewallet\/walletexception\/failurereason","kind":"symbol","abstract":[],"identifier":"doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletException\/failureReason","type":"topic","role":"symbol","title":"failureReason","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"failureReason","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":"?","kind":"text"}]},"doc://com.pingidentity.PingOneWallet/documentation/PingOneWallet/WalletException/recoverySuggestion":{"identifier":"doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletException\/recoverySuggestion","url":"\/documentation\/pingonewallet\/walletexception\/recoverysuggestion","role":"symbol","kind":"symbol","title":"recoverySuggestion","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"text":"recoverySuggestion","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"abstract":[],"type":"topic"},"doc://com.pingidentity.PingOneWallet/documentation/PingOneWallet/WalletException/helpAnchor":{"url":"\/documentation\/pingonewallet\/walletexception\/helpanchor","type":"topic","title":"helpAnchor","role":"symbol","abstract":[],"identifier":"doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletException\/helpAnchor","kind":"symbol","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"helpAnchor"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"kind":"text","text":"?"}]},"doc://com.pingidentity.PingOneWallet/documentation/PingOneWallet":{"title":"PingOneWallet","identifier":"doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet","kind":"symbol","role":"collection","url":"\/documentation\/pingonewallet","abstract":[{"text":"This defines the usage and interfaces of the PingOne Wallet SDK for iOS to interact with PingOne credentialing services. The SDK provides for a user to receive, save, and share credentials for identification and verification purposes.","type":"text"}],"type":"topic"}}}